tests/ui/mut_range_bound.rs

```diff
@@ -8,6 +8,9 @@ fn main() {
     immut_range_bound();
     mut_borrow_range_bound();
     immut_borrow_range_bound();
+    mut_range_bound_break();
+    mut_range_bound_nested_break();
+    multiple_mut_range_bound_break();
```

Comment:
Not sure why this needs to be here

Reply:
Back in the day those were necessary to suppress `unused` warnings. Now those warnings are disabled by default.

Long story short: Yes you can remove them.